backend/routes/news.js

```diff
@@ -17,29 +17,43 @@ router.get("/", async (req, res) => {
   }
 });
 
+// user does not access this? For now, will access when user input is considered in seeded data (maybe?)
 router.post("/seed-news", async (req, res) => {
+  const newsData = [];
   const apiKey = process.env.API_KEY;
-  const response = await fetch(
-    `https://newsapi.org/v2/top-headlines?country=us&apiKey=${apiKey}&language=en`,
-  );
-  const data = await response.json();
-  const articles = data.articles;
 
-  console.log(articles);
+  const Categories = {
+    // enum
+    BUSINESS: "business",
+    TECH: "technology",
+    ENTERTAINMENT: "entertainment",
+    GENERAL: "general",
+    HEALTH: "health",
+    SCIENCE: "science",
+    SPORTS: "sports",
+  };
 
-  // news to add
-  const newsData = [];
-  for (const article of articles) {
-    const newArticle = {
-      name: article.title,
-      releaseDate: article.publishedAt,
-      category: "latest", // hardcoded for now until category is found
-      articleURL: article.url,
-      imageURL: article.urlToImage,
-      userId: req.session.userId,
-    };
-
-    newsData.push(newArticle);
+  // optimize? (O^2 time complexity)
+  for (const category of Object.values(Categories)) {
+    const response = await fetch(
+      `https://newsapi.org/v2/top-headlines?country=us&apiKey=${apiKey}&language=en&category=${category}`,
```

Comment:
1. Do articles ever have multiple categories attached? 
2. Do we need to fetch articles for each category and then push them into newsData? Or can we just do a general fetch for all categories?